run-ci.sh

```diff
@@ -29,8 +29,11 @@ else
 fi
 
 if [ ${RUN_TEST=1} -eq 1 ]; then
+    rm -f tests/loggedkeys.txt
```

Comment:
I think rather than having it at this level, we should bring it lower, as we've seen it can have an impact on `dub`. Either nesting the invocation or running a preGenerate / postRun script ?

Reply:
Done